### ISSUE TITLE
Please use interpreterPath instead pythonVenv

### ISSUE THREAD

**Issue by danielniccoli** [microsoft/vscode-azurefunctions#3828]
The extension should use the configured interpreter path `python.interpreterPath` instead of an addition custom variable `azureFunctions.pythonVenv`.

**Comment by alexweininger** [comment]
This sounds like a good suggestion. In my Python settings I see a `python.venvPath` setting? Did you mean to suggest that?

**Comment by danielniccoli** [comment]
No, I mean `python.interpreterPath.` Why should I have to configure a setting in a file, when I can use the interpreterPath, which is set per project anyway?

**Comment by nturinski** [comment]
I was investigating this one, and there's an annoying issue as to why this won't work for us.

It seems like if `python.interpreterPath` has a space in it, it won't actually run the python command for us.

You end up with something like this:

![image](https://github.com/microsoft/vscode-azurefunctions/assets/5290572/869f6bf9-6ba5-4d20-a73c-8541efff04ec)

Works fine without a space though.

![image](https://github.com/microsoft/vscode-azurefunctions/assets/5290572/6fcf31e0-daf6-4c0c-8e42-569da822ddfc)

This might be a windows only issue for now. I'm trying to see if there's a way for me to get the interpreter in our runtime code so that we can convert it, but I think it'd be better if the Python team just wrapped their fs path with quotations.

I have an issue tracking it here: https://github.com/microsoft/vscode-python/issues/23028
